test/pytest/test_samples.py::test_descriptions started to fail in F41/42

### Explain what happens

The copr RPM builds still worked in #22081, but upon landing it, the [run on main](https://github.com/cockpit-project/cockpit/commit/f5c1bdcba72984265569cb6ab20a06f966c2c4ae) fails:

```
______________________________ test_descriptions _______________________________

    def test_descriptions():
        for cls in cockpit.samples.SAMPLERS:
            # currently broken in containers with no cgroups or temperatures present
            if cls in [cockpit.samples.CGroupSampler, cockpit.samples.CPUTemperatureSampler]:
                continue
    
>           get_checked_samples(cls())

test/pytest/test_samples.py:109: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

sampler = <cockpit.samples.MountSampler object at 0x7faf2246ae40>

    def get_checked_samples(sampler: cockpit.samples.Sampler) -> cockpit.samples.Samples:
        cls = sampler.__class__
    
        samples: cockpit.samples.Samples = collections.defaultdict(dict)
        sampler.sample(samples)
    
>       assert set(samples) == {descr.name for descr in cls.descriptions}
E       AssertionError: assert set() == {'mount.total', 'mount.used'}
E         
E         Extra items in the right set:
E         'mount.total'
E         'mount.used'
E         
E         Full diff:
E         + set()...
E         
E         ...Full output truncated (4 lines hidden), use '-vv' to show

test/pytest/test_samples.py:90: AssertionError
```

CentOS 9/10 works. This smells like some bodhi update just landed. Not yet reproducible locally with `dnf update`, and I went through recent bodhis and nothing stands out. I suppose in a few days it'll be easier to reproduce once packages hit the mirrors?

### Version of Cockpit

_No response_

### Where is the problem in Cockpit?

None

### Server operating system

None

### Server operating system version

_No response_

### What browsers are you using?

_No response_

### System log

```shell

```

--- a/test/pytest/test_samples.py
+++ b/test/pytest/test_samples.py
@@ -102,8 +102,9 @@ def get_checked_samples(sampler: cockpit.samples.Sampler) -> cockpit.samples.Sam
 
 def test_descriptions():
     for cls in cockpit.samples.SAMPLERS:
-        # currently broken in containers with no cgroups or temperatures present
-        if cls in [cockpit.samples.CGroupSampler, cockpit.samples.CPUTemperatureSampler]:
+        # currently broken in containers with no cgroups or temperatures or mounts present
+        if cls in [cockpit.samples.CGroupSampler, cockpit.samples.CPUTemperatureSampler,
+                   cockpit.samples.MountSampler]:
             continue
 
         get_checked_samples(cls())

